features/facebookImport/test/ministory/importing-status-analysys.test.js

```diff
@@ -1,4 +1,4 @@
-import DataImportingStatusAnalysis from "../../src/model/analyses/report/importing-status-analysis";
+import { DataImportingStatusAnalysis } from "@polypoly-eu/poly-analysis";
```

Comment:
Thanks for catching this :) now I am running to another dilema: 
The Report Ministories are called reports so when we import them they are called "[nameOfAnalysis]Report" e.g. "DataImportingStatusReport" so naming the analysis report goes against the convention that we had in facebookImport. 
It makes sense to make a distinction between reportsAnalyses and analyses which is why they are of class ReportAnalysis, but naming them reports will cause a larger change in how they are refered to. Though I don't mind the rename, for all intents and purposes they are treated as analyses (they are still triggered by analyzeFile in the overview). So if you don't have a strong preference I will revert back to calling them Analysis to keep the Report suffix for the views (ReportStories). 
Otherwise the View reports will have to be renamed to "[nameOfAnalysis]ReportStory" instead of "[nameOfAnalysis]Report".